pkg/virt-controller/watch/application.go

```diff
@@ -264,16 +264,19 @@ func (vca *VirtControllerApp) getNewRecorder(namespace string, componentName str
 func (vca *VirtControllerApp) initCommon() {
 	var err error
 
-	containerdisk.SetLocalDirectory(vca.ephemeralDiskDir + "/container-disk-data")
+	virtClient, err := kubecli.GetKubevirtClient()
 	if err != nil {
 		golog.Fatal(err)
 	}
+
+	containerdisk.SetLocalDirectory(vca.ephemeralDiskDir + "/container-disk-data")
```

Comment:
Good catch, I think it's an artefact of a rebase to master.

Reply:
Actually, not really: previously, the check was not effective since it was just checking that the default value of the variable is nil (it is). There is no code called that would override the default value. Now, I effective reuse the previously redundant error code check. I don't think a change is needed here.

Reply:
yes I saw, this was always on this form you know why they did something like this? it always going to be nil.

Reply:
but also unrelated to the PR